File: PROJECT/Models/DTOs/Shipping/Econt/CityDTO.cs
namespace Models.DTOs.Shipping.Econt
{
    public class CityDTO
    {
        public CountryDTO? Country { get; set; }
        public string? Name { get; set; }
        public string? PostCode { get; set; }
        public string? NameEn { get; set; }
        public string? RegionName { get; set; }
        public string? RegionNameEn { get; set; }
        public string? PhoneCode { get; set; }
        public LocationDTO? Location { get; set; }
        public bool? ExpressCityDeliveries { get; set; }
        public bool? Monday { get; set; }
        public bool? Tuesday { get; set; }
        public bool? Wednesday { get; set; }
        public bool? Thursday { get; set; }
        public bool? Friday { get; set; }
        public bool? Saturday { get; set; }
        public bool? Sunday { get; set; }
        public int? ServiceDays { get; set; }
        public int? ZoneId { get; set; }
        public string? ZoneName { get; set; }
        public string? ZoneNameEn { get; set; }
    }
}

File: PROJECT/FRONTEND/Common/DTOs/CityDTO.ts


export class CityDTO { 
    constructor(obj: Partial<CityDTO>) {
        Object.assign(this,obj);
    }

    
    public country: CountryDTO;
    public name: string | null;
    public postCode: string | null;
    public nameEn: string | null;
    public regionName: string | null;
    public regionNameEn: string | null;
    public phoneCode: string | null;
    public location: LocationDTO;
    public expressCityDeliveries: boolean | null;
    public monday: boolean | null;
    public tuesday: boolean | null;
    public wednesday: boolean | null;
    public thursday: boolean | null;
    public friday: boolean | null;
    public saturday: boolean | null;
    public sunday: boolean | null;
    public serviceDays: number | null;
    public zoneId: number | null;
    public zoneName: string | null;
    public zoneNameEn: string | null;
}
